bin/build-plugin-zip.sh

```diff
@@ -73,9 +73,7 @@ build_files=$(ls dist/*/*.{js,css})
 status "Creating archive... 🎁"
 zip -r woocommerce-admin.zip \
 	woocommerce-admin.php \
-	lib/*.php \
-	includes/*.php \
-	includes/**/*.php \
+	includes/ \
 	images/* \
```

Comment:
No need to change it in this PR, but I think we can get rid of this asterisk.
```suggestion
	images/ \
```